src/dotnet/APIView/APIViewWeb/Helpers/LanguageServiceHelpers.cs

```diff
@@ -44,6 +44,28 @@ public static string MapLanguageAlias(string language)
         public static LanguageService GetLanguageService(string language, IEnumerable<LanguageService> languageServices)
         {
             return languageServices.FirstOrDefault(service => service.Name == language);
-        } 
+        }
+
+        public static bool UseLineBreakForParameterSeparator(string language)
+        { 
+            switch(language)
+            {
+                case "Python":
+                    return true;
+                default:
+                    return false;
+            }
+        }
+
+        public static bool UsesTreeStyleParser(string language)
+        {
+            switch (language)
+            {
+                case "C#":
```

Comment:
I would prefer to have this as a property in Langauge service class for each langauge instead of a helper method. Same is the case for above method too to keep it simple instead of a hardcoded gate function to enable a feature at langauge level.